### PR TITLE
Updated ops for new sample loader

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context

Ops tool has been updated to use the latest changes from the sample loader

# What has changed

- Updated to latest version of sample loader(Need to change to release version once sample loader is merged)
- updated tests
# How to test?

- Run with other branches on ticket.
- Try loading a sample and trigger action rules, all should work.

# Links

[Trello](https://trello.com/c/zm2Ssvnn/771-make-sample-load-changes-to-meet-new-sample-spec-13)

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[Make sample load changes to meet new sample spec (13)](https://trello.com/c/zm2Ssvnn/771-make-sample-load-changes-to-meet-new-sample-spec-13)

File: Pipfile [hunk 1]
```diff
@@ -4,17 +4,17 @@ verify_ssl = true
...
 gunicorn = "*"
 flask-httpauth = "*"
 flask-assets = "*"
 pika = "*"
 redis = "*"
-werkzeug = "<0.15"
+werkzeug = "*"
 census-rm-qid-batch-runner = {editable = true,git = "https://github.com/ONSdigital/census-rm-qid-batch-runner",ref = "v1.4.0"}
-census-rm-sample-loader = {editable = true,git = "https://github.com/ONSdigital/census-rm-sample-loader",ref = "v1.4.0"}
+census-rm-sample-loader = {editable = true,git = "https://github.com/ONSdigital/census-rm-sample-loader",ref = "771-make-sample-load-changes-to-meet-new-sample-spec"}
```
Comment (NickGrantX) [inline Pipfile]: obvs needs to change to v1.5.0 before merging

**Review by NickGrantX — APPROVED** [review]
ATs passing. LGTM 👍

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 language: python
-python: '3.6'
+python: '3.7'
```
Comment (AdamHawtin) [inline .travis.yml:2]: 👍 👍 👍 
(worth discussing sometime when we start using 3.8 too)

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**